SPECS/mlnx-ofa_kernel/mlnx-ofa_kernel.spec

```diff
@@ -25,133 +25,91 @@
 # and/or other materials provided with the distribution.
 #
 #
-
+Summary:        Infiniband HCA Driver
+Name:           mlnx-ofa_kernel
+Version:        23.04
+Release:        3%{?dist}
```

Comment:
since you've bumped the version, you can reset the release number back to 1

Reply:
Done